lblod/frontend-lokaal-mandatenbeheer
# Add fold fracties component to mandatarissen search

## Description

Also add folding to the mandatarissen search page, in the same way as in organen and mandataris detail pages.

## How to test
- go to mandatarissen/search
- see that there are folded fracties like `Onafhankelijk (fractie-name)`
- check in the detail of the persoon if these are the correct fracties for the selected bestuursperiode
